### ISSUE TITLE
documenting a c++ function with unnammed parameter raise an exception

### ISSUE THREAD

**Issue by raiseFlaymeException** [kkoomen/vim-doge#648]
**Describe the bug**

**Settings**

(on neovim 0.10)

(init.lua)
```neovim
vim.g.doge_enable_mappings = 0
```

(after/doge.lua)
```neovim
vim.g.doge_doc_standard_cpp = "doxygen_cpp_comment_slash"

vim.keymap.set('n', '<leader>d', ':DogeGenerate<Enter>')

vim.keymap.set('n', '<Tab>', '<Plug>(doge-comment-jump-forward)')
vim.keymap.set('n', '<S-TAB>', '<Plug>(doge-comment-jump-backward)')
vim.keymap.set('i', '<TAB>', '<Plug>(doge-comment-jump-forward)')
vim.keymap.set('i', '<S-TAB>', '<Plug>(doge-comment-jump-backward)')
vim.keymap.set('s', '<TAB>', '<Plug>(doge-comment-jump-forward)')
vim.keymap.set('s', '<S-TAB>', '<Plug>(doge-comment-jump-backward)')
```

**To Reproduce**

1. :DogeGenerate on a function with unnammed parametter like:

```cpp
Game(const Game &)=default;
```

**Expected behavior**

somethings like:

```
///
/// @brief [TODO:summary]
///
/// @param[const Game &] [TODO:description]
///
```

**Screenshots**

![bug_doge](https://github.com/kkoomen/vim-doge/assets/126109305/9cb39336-9792-4ceb-89f7-7b312895d67d)

**Output of `./bin/vim-doge-helper --version`**

vim-doge-helper 4.6.0

**Output of `vim --version`**

nvim --version :

```
NVIM v0.10.0-dev-64c2c7c
Build type: RelWithDebInfo
LuaJIT 2.1.1699801871
Run "nvim -V1 -v" for more info
```

nvim -V1 -v :

```
NVIM v0.10.0-dev-64c2c7c
Build type: RelWithDebInfo
LuaJIT 2.1.1699801871
Compilation: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x64/cl.exe /DWIN32 /D_WINDOWS /MD /Zi /O2 /Ob1  -W3 -wd4311 -wd4146 -wd4003 -wd4715 -wd4003 -DUNIT_TESTING -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_DEPRECATE -D_WIN32_WINNT=0x0602 -DMSWIN -DINCLUDE_GENERATED_DECLARATIONS -DNVIM_TS_HAS_SET_MAX_START_DEPTH -ID:/a/neovim/neovim/.deps/usr/include/luajit-2.1 -ID:/a/neovim/neovim/.deps/usr/include -ID:/a/neovim/neovim/build/src/nvim/auto -ID:/a/neovim/neovim/build/include -ID:/a/neovim/neovim/build/cmake.config -ID:/a/neovim/neovim/src

         fichier vimrc système : "$VIM\sysinit.vim"                    <translated: file vimrc system: : "$VIM\sysinit.vim">
               $VIM par défaut : "C:/Program Files (x86)/nvim/share/nvim"   <translated: $VIM by default "C:/Program Files (x86)/nvim/share/nvim">
```

Run :checkhealth for more info

PS:
sorry for my english and for creating a lot of issue (see my feature called "Add automaticly the type for the comment for c++").

thank for more time,

Flayme

**Comment by kkoomen** [comment]
This bug has been fixed and released in [v4.6.1](https://github.com/kkoomen/vim-doge/releases/tag/v4.6.1).

Feel free to submit any new issues if you experience any unwanted behavior in the future. Thanks for your contribution.

**Comment by raiseFlaymeException** [comment]
thank you that works now
